### PR TITLE
Feat : Added Playlist management

### PR DESCRIPTION
**Pull request by srejitk**
The following merge request adds the following features : 

1. Playlist Page
2. Single Playlist Page
3. Modal Component

### CODE REVIEW

File: src/components/HorizontalVideoCard/HorizontalVideoCard.module.css [hunk 1]
```diff
@@ -107,3 +108,104 @@
...
+    right: 0.25rem;
+  }
+
+  .thumbnail {
+    transform: scale(1.3);
+  }
+
+  .preview {
+    display: none;
+    transform: scale(1.3);
```
Comment (nishasen) [inline src/components/HorizontalVideoCard/HorizontalVideoCard.module.css:165]: u can remove the repeatitive code

**Review by nishasen — APPROVED** [review]
Code looks good, some of the cpde is repeatitive U can remove those

**Review by sadathshariff — APPROVED** [review]
The code looks clean and understandable, 
Looks good to me

File: src/components/HorizontalVideoCard/HorizontalVideoCard.module.css (hunk 1 above)
Comment (srejitk) [reply src/components/HorizontalVideoCard/HorizontalVideoCard.module.css:165]: Will do. Thank You.

**Pull request merged**